definitions/variable/energy/energy-final.yaml

```diff
@@ -19,409 +19,361 @@
     energy use: all excl. losses
     sector: all end-users
     unit: [EJ/yr, GWh]
-- Final Energy|Electricity:
-    description: Final energy consumption of electricity (including on-site solar pv),
-      excluding transmission/distribution losses
-    energy source: electricity
-    energy use: all excl. losses
-    sector: all end-users
-    unit: [EJ/yr, GWh]
-- Final Energy|Fossil:
-    description: Final energy consumption from fossil fuels
-    energy source: fossil fuels
+- Final Energy (w/o bunkers):
+    description: Total final energy consumption by all end-use sectors and all fuels,
+      excluding transmission/distribution losses, excluding bunker fuels, excluding pipelines 
+    energy source: all
     energy use: all excl. losses
     sector: all end-users
     unit: [EJ/yr, GWh]
```

Comment:
I think we had this discussion before, maybe it didn't included you at the time. 
If my memory does not fail me, it was decided that multiple units are allowed, however at each project all result submissions should only upload results using a single unit standard to avoid any issues.

Reply:
Ah right I see, I assume that this has to do with the definitions being used in openENTRANCE as well as ECEMF?
This approach is fine, however it is a little dangerous as there is no safeguard against mixed unit uploads. 
Maybe we want to implement a project specific check in `workflow.py`.